README.md

```diff
@@ -293,7 +293,7 @@ These are the available config options for making requests. Only the `url` is re
   },
 
   // `data` is the data to be sent as the request body
-  // Only applicable for request methods 'PUT', 'POST', and 'PATCH'
+  // Only applicable for request methods 'PUT', 'POST', 'DELETE , and 'PATCH'
```

Comment:
I think "applicable" here is something like "suggested but not disabled". And hope you can clarify for `transformRequest` as well.

Reply:
```suggestion
  // Only applicable(suggested but not forbidden) for request methods 'PUT', 'POST', 'DELETE , and 'PATCH'
```

Reply:
I don't think it requires expansion